opensubtitles/discover.go

```diff
@@ -0,0 +1,51 @@
+package opensubtitles
+
+import (
+	"context"
+	"net/http"
+)
+
+type DiscoverService service
```

Comment:
Codacy found an issue: [exported type DiscoverService should have comment or be unexported](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)